appwidget-configuration/src/main/java/com/google/android/glance/appwidget/configuration/AppWidgetConfigurationScaffold.kt

```diff
@@ -160,17 +161,23 @@ class AppWidgetConfigurationState(
  */
 @Composable
 fun rememberAppWidgetConfigurationState(configurationInstance: GlanceAppWidget): AppWidgetConfigurationState {
+    val appWidgetId = (LocalContext.current as Activity).intent?.extras?.getInt(
```

Comment:
Instead of redoing the logic again use the getAppWidgetId method
https://developer.android.com/reference/kotlin/androidx/glance/appwidget/GlanceAppWidgetManager#getAppWidgetId(androidx.glance.GlanceId)

So bring the `val glanceId` below up and then use the `getAppWidgetId` method to get the ID.

Reply:
Has that thought as well. 👍